IDesign/IDesign.CommonResources/ResourceUtils.cs

```diff
@@ -31,15 +32,21 @@ public static string GetResourceFromString(string name)
             return ResourceManager.GetString(name, Culture);
         }
 
-        public static string ResourceMessageToString(IResourceMessage ResMessage)
+        public static string ResourceMessageToString(IResourceMessage resMessage, ICheckResult result)
```

Comment:
This will break stuff, keep the orignal syntax working by giving a default value for result